android/views/src/main/kotlin/com/boswelja/ephemeris/views/EphemerisCalendarView.kt

```diff
@@ -26,6 +26,14 @@ class EphemerisCalendarView @JvmOverloads constructor(
 
     private var pagingDataSource: CalendarPageLoader? = null
 
+    var focusMode: FocusMode? = null
```

Comment:
I'd prefer a default state over null so that users can just setup the calendar without **having** to set its values. I'll also add XML attributes for all these.

Reply:
Sounds good, we can set the default to just have all days in focus for now